### ISSUE TITLE
Take files from email and upload into OAB for a request

### ISSUE THREAD

**Issue by JosephMcArthur** [OAButton/discussion#1017]
@markmacgillivray so you've said that if we can have files with attachements we want uploaded into the system sent to a "system address" you could take that file and upload it to the relavent request.

Is that the case?

**Comment by markmacgillivray** [comment]
Yes, we could. There are various ways. Simplest would be to use a gmail account with IFTTT or some other script that sends it on to our server.

Note though that this means either the authors are responding to a system address, or the authors still respond to Natalia (or some other admin) and then Natalia sends it on to the system address. So Natalia would still have to do something, unless you want to go back to the authors receiving contacts direct from the system.

**Comment by JosephMcArthur** [comment]
I have a gmail filter set up that fairly accurate picks up emails with papers attached, and can be used to forward them to any email. In those threads is also always a request ID or author email address that could be used to match it up to a request :D
